documentation/api/io.strimzi.api.kafka.model.KafkaBridgeConsumerSpec.adoc

```diff
@@ -38,9 +36,9 @@ spec:
     config:
       auto.offset.reset: earliest
       enable.auto.commit: true
-      ssl.cipher.suites: TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
-      ssl.enabled.protocols: TLSv1.2
-      ssl.protocol: TLSv1.2
-      ssl.endpoint.identification.algorithm: HTTPS
     # ...
 ----
+
+IMPORTANT: The Cluster Operator does not validate keys or values in the `config` object.
+If an invalid configuration is provided, the Kafka Bridge cluster might not start or might become unstable.
```

Comment:
Why we say "cluster" here. I think it's misleading. Wouldn't it be better just "deployment" instead?